pkg/services/rpcsrv/server_test.go

```diff
@@ -73,30 +73,70 @@ type rpcTestCase struct {
 	check   func(t *testing.T, e *executor, result any)
 }
 
-const genesisBlockHash = "0f8fb4e17d2ab9f3097af75ca7fd16064160fb8043db94909e00dd4e257b9dc4"
-const testContractHash = "449fe8fbd4523072f5e3a4dfa17a494c119d4c08"
-const deploymentTxHash = "bbb8ec059dd320dc9de5a8fb8c75351d8e369fca0256f7a6bdb623dcf71861ed"
-
+// Constants and variables below are taken from the logs generated by
+// TestCreateBasicChain. These constants and variables should be updated manually
```

Comment:
`TestCreateBasicChain` is not from this package, should we somehow try to point out what exact func is it?

Reply:
I thought about it, OK, let’s do.